sdk/communication/azure-communication-identity/azure/communication/identity/_shared/models.py

```diff
@@ -14,8 +14,11 @@ class DeprecatedEnumMeta(CaseInsensitiveEnumMeta):
 
     def __getattribute__(cls, item):
         if item.upper() == "MICROSOFT_BOT":
-            warnings.warn("MICROSOFT_BOT is deprecated and has been replaced by \
-                          MICROSOFT_TEAMS_APP identifier.", DeprecationWarning)
+            warnings.warn(
+                "MICROSOFT_BOT is deprecated and has been replaced by \
+                          MICROSOFT_TEAMS_APP identifier.",
+                DeprecationWarning,
```

Comment:
```suggestion
                DeprecationWarning
```
I guess here comma is not necessary

Reply:
That was added by Black not me, I don't think I can alter it.